tests/global-setup.js

```diff
@@ -7,5 +7,5 @@
  * Global test setup used to configure process environment.
  */
 export function setup() {
-	process.env.TZ = 'UTC'
+	globalThis.process.env.TZ = 'UTC'
```

Comment:
Why?

Reply:
> Silencing

As this is a configuration issue, not a problem in the code, we can silence it by `// eslint-ignore-next-line`